cogs/wiki.py

```diff
@@ -100,7 +100,6 @@ async def wiki_helper(self, limit: Optional[int], ctx: Context, *args):
                   'search': query,
                   'namespace': get_wiki_namespaces('Main,Category,Modding'),
                   'limit': self.title_limit if limit is None else limit,
-                  'profile': 'fuzzy',
```

Comment:
Hmm, actually - see now that this is indeed a supported parameter, as outlined here: https://www.mediawiki.org/wiki/API:Opensearch

I was fooled because this parameter does not show up in the CoQ wiki API sandbox (https://cavesofqud.gamepedia.com/Special:ApiSandbox#action=opensearch&format=json&search=mayor%20haddas&namespace=0%7C14%7C10000&limit=1&redirects=resolve)

The reason I removed this is because an API search with limit=1 for "Mayor Haddas", "mayor haddas", or "Mayor haddas" always returned the page "Mayor Haddas/Conversations" instead of "Mayor Haddas". I was troubleshooting trying to figure out why, and I noticed that when I remove this parameter, it started working "correctly".

However, this is somewhat of an edge case - I didn't notice search inaccuracies in any other scenarios, so we can restore this if it's important

Reply:
Well, now it works 🤔 so, nevermind. I put it back 😄